tests/apps/backups/diff/DiffFilesCalculator.test.ts

```diff
@@ -50,7 +52,73 @@ describe('DiffFilesCalculator', () => {
     const { modified } = DiffFilesCalculator.calculate(local, remote);
 
     expect(modified.size).toBe(expectedNumberOfFilesToModify);
-    expect(Array.from(modified.keys())).toStrictEqual(local.files);
-    expect(Array.from(modified.values())).toStrictEqual(remote.files);
+  });
+
+  it('identifies unmodified files correctly', () => {
+    const expectedNumberOfUnmodifiedFiles = 10;
+    const local = LocalTreeMother.oneLevel(expectedNumberOfUnmodifiedFiles);
+    const remote = RemoteTreeMother.cloneFromLocal(local);
+
+    const { unmodified, added, deleted, modified } =
+      DiffFilesCalculator.calculate(local, remote);
+
+    expect(unmodified.length).toBe(expectedNumberOfUnmodifiedFiles);
+    expect(added.length).toBe(0);
+    expect(deleted.length).toBe(0);
+    expect(modified.size).toBe(0);
+  });
+
+  it('handles mixed additions, deletions, modifications, and unmodified files', () => {
+    const expectedNumberOfFilesToAdd = 5;
+    const expectedNumberOfFilesToDelete = 5;
+    const local = LocalTreeMother.oneLevel(10);
+    const remote = RemoteTreeMother.cloneFromLocal(local);
+
+    for (let i = 0; i < 5; i++) {
```

Comment:
This seems like needing a fixture that generates a bunch of: 
```
LocalFileMother.fromPartial({
  path: AbsolutePathMother.anyFile(),
  modificationTime: DateMother.today().getTime(),
})
```
So you do not have to use for loops inside a test